vectordb_bench/backend/clients/chroma/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import SecretStr
+from ..api import DBConfig
+
+class ChromaConfig(DBConfig):
+    name: str = ""
+    password: str = ""
+    host: str = ""
```

Comment:
Please use Pydantic SecretStr Type, DB config will be recorded in results, we don't want any sensitive infomations exposed to everyone.
ref:  https://github.com/zilliztech/VectorDBBench/blob/675b48f0d2b5561886af17dd897d789bd5e76f5b/vectordb_bench/backend/clients/milvus/config.py#L6